docs/source/index.rst

```diff
@@ -3,18 +3,58 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to HookMan's documentation!
-===================================
+=================================
+HookMan documentation!
```

Comment:
`!` is not needed (perhaps neither "documentation")

Reply:
That was generated by `sphinx-quickstart`, and I agree that they are not necessary

Reply:
Fixed ✔️